lib/simctl.ts

```diff
@@ -120,11 +121,19 @@ export class Simctl implements ISimctl {
 		return devices;
 	}
 
+	public getLog(deviceId: string): any {
+		return this.simctlSpawn("spawn", [deviceId, "log", "stream"]);
```

Comment:
You should add these:
`--predicate 'senderImagePath contains "NativeScript"'`   <-- This filters it for you to just NativeScript apps.
`--style syslog `  <-- This gives you a log very similar to what you are already consuming.

Reply:
Thank you for the great suggestions, NathanaelA!

I will add `--style syslog` as this will really make the log very similar to the one in Simulators with previous iOS versions.

I can't add `--predicate 'senderImagePath contains "NativeScript"` directly as this tool is not framework specific and is published as separate NPM package. I can however add an optional predicate argument to the method.